global.json

```diff
@@ -1,6 +1,6 @@
 {
     "sdk": {
-        "version": "6.0.100",
+        "version": "6.0.300",
```

Comment:
We're in the process of taking this change in another change. Do you need 6.0.300 in order for the changes to work? In any case, you should exclude this from the PR, even if locally you're using 6.0.300 for testing.

Reply:
I forgot to change back when troubleshooting job submission issues. Changes will work with 6.0.100